package.json

```diff
@@ -0,0 +1,96 @@
+{
+  "name": "flow-diagram-explorer",
```

Comment:
@rubenthoms @jsolaas You can decide this later of course, but could perhaps be natural to put the npm package under the Equinor scope? https://www.npmjs.com/org/equinor